### ISSUE TITLE
Duplicates animals by teleport

### ISSUE THREAD

**Issue by Kuupo** [RobotoRaccoon/StableMaster#15]
Greeting!
There seem to be a way to duplicate animals upon teleport, not sure if its Minecraft itself.

1. Select a horse to tp
2. tp yourself to a wished destination (to an unloaded chunk).
3. /sm tp, to teleport the horse to you, message "teleporting..." shows up, but no horse to see.
4. tp yourself back to where the horse was in the beginning, horse is still there.
5. tp yourself back to where you wished the horse was suppose to teleport to, horse is then there as well, so now there is 1 horse on each places.

But if there is a player where the horse start location is and stays there while another player does the /sm tp in a different place, the horse wont duplicate and it will teleport. I assume it might be something with animals in unloaded chunks.

Server version: paper 1.14.3
StableMaster version 1.1.5

**Comment by RobotoRaccoon** [comment]
Yeah, this is a weird issue thanks to 1.14 and its new asynchronous chunk loading. I've got it to a state when it'll properly error when a chunk isn't loaded, but I can't get it to load the chunks for the TP.
Users will have to most likely teleport animals in pairs (one person to stand with the animal whilst it's teleporting).
Leaving this open, hopefully a better fix to come...
